src/1_create_canonical.jl

```diff
@@ -136,8 +135,8 @@ string_cols = contains.(string.(typeof.(eachcol(output_features))), "String")
 output_features[!, contains.(string.(typeof.(eachcol(output_features))), "String")] .= String.(output_features[:, string_cols])
 
 # Save geopackage
-GDF.write(joinpath(OUTPUT_DIR, "rrap_shared_lookup_$(today()).gpkg"), output_features; crs=GFT.EPSG(4326))
+GDF.write(joinpath(OUTPUT_DIR, "rrap_shared_lookup_$(Dates.format(now(),"YYYY-mm-dd-THH-MM")).gpkg"), output_features; crs=GFT.EPSG(4326)) #removed time so name for next script is always the same
```

Comment:
Standard is to have two spaces after code then to have the comment.

But aside from that, the comment says time has been removed, but the date format seems to include time (`THH-MM`)?

```suggestion
GDF.write(joinpath(OUTPUT_DIR, "rrap_shared_lookup_$(Dates.format(now(),"YYYY-mm-dd-THH-MM")).gpkg"), output_features; crs=GFT.EPSG(4326))  # removed time so name for next script is always the same
```